ppr-ui/src/interfaces/mhr-registration-interfaces/HomeSectionIF.ts

```diff
@@ -0,0 +1,8 @@
+// New mhr registration Home Section interface.
+export interface HomeSectionIF {
```

Comment:
Just an observation, I think inches will have a default value of 0.

Reply:
Interesting, we could definitely initialize store with `0`!

I'll reach out to UX if that is also a default they would expect in the UI, my assumption is that they would prefer to prompt a user to input a value using the usual validation prompts.

Thank you for the heads up!